### PR TITLE
feat: Add new output called env_vars for postgresql

### PR DESCRIPTION
**Pull request by q2w**
This PR makes below changes,

1. Add a new output variable called env_vars to postgresql
2. Updates the metadata.yaml file
3. Update test for the new env_vars output

### CODE REVIEW

File: examples/postgresql-public/outputs.tf [hunk 1]
```diff
@@ -38,3 +38,8 @@ output "public_ip_address" {
   description = "The first public (PRIMARY) IPv4 address assigned for the master instance"
   value       = module.postgresql-db.public_ip_address
 }
+
+output "output_env_vars" {
```
Comment (bharathkkb) [inline examples/postgresql-public/outputs.tf:42]:
```suggestion
output "env_vars" {
```

File: modules/postgresql/main.tf [hunk 2]
```diff
@@ -65,12 +67,13 @@ resource "google_sql_database_instance" "default" {
...
-    availability_type           = var.availability_type
-    deletion_protection_enabled = var.deletion_protection_enabled
-    connector_enforcement       = local.connector_enforcement
+    tier                         = var.tier
+    edition                      = var.edition
+    activation_policy            = var.activation_policy
+    availability_type            = var.availability_type
+    deletion_protection_enabled  = var.deletion_protection_enabled
+    connector_enforcement        = local.connector_enforcement
+    enable_google_ml_integration = var.enable_google_ml_integration
```
Comment (bharathkkb) [inline modules/postgresql/main.tf:76]: Let's split unrelated changes like this and below to a separate PR

File: test/integration/postgresql-public/postgresql_public_test.go [hunk 3]
```diff
@@ -63,6 +63,12 @@ func TestPostgreSqlPublicModule(t *testing.T) {
 		assert.Equal(2, len(ipAddresses), "Expected 2 addresses")
 		assert.Equalf(pSql.GetStringOutput("public_ip_address"), ipAddresses[0].Get("ipAddress").String(), "Expected %s PublicIp", pSql.GetStringOutput("public_ip_address"))
 		assert.Equal("PRIMARY", ipAddresses[0].Get("type").String())
+
+		// assert env_vars output
+		envVars := pSql.GetStringOutput("output_env_vars")
```
Comment (bharathkkb) [inline test/integration/postgresql-public/postgresql_public_test.go:68]:
```suggestion
		envVars := pSql.GetStringOutput("env_vars")
```

**Review by bharathkkb — CHANGES_REQUESTED** [review]

File: modules/postgresql/main.tf (hunk 2 above)
Comment (q2w) [reply modules/postgresql/main.tf]: Done!

**Review by bharathkkb — APPROVED** [review]
